### PR TITLE
Change pen color whiteboard while reviewing

### PR DESCRIPTION
**Pull request by infinyte7**
## Purpose / Description
While reviewing there is need of different pen color for different answer for better retention.

## Fixes
Fixes _Link to the issues._

## Approach
1. Added a xml containing six button as color background. 
2. When whiteboard turn on then one option ```Pen Color``` also visible
3. Selecting respective color button change pen color

## How Has This Been Tested?
Tested on Emulator
1. When Whiteboard turned on
2. Select pen color 
3. Draw in that color

pen

## Learning (optional, can help others)
_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

## Checklist
_Please, go through these checks before submitting the PR._

- [ x ] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [ x ] You have a descriptive commit message with a short title (first line, max 50 chars).
- [ x ] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [ x ] You have commented your code, particularly in hard-to-understand areas
- [ x ] You have performed a self-review of your own code

### CODE REVIEW

**Comment by infinyte7** [comment]
Why there is conflicts?

**Comment by infinyte7** [comment]
pen_color

File: AnkiDroid/src/main/java/com/ichi2/anki/Reviewer.java [hunk 1]
```diff
@@ -99,6 +107,48 @@ protected int getToastResourceId() {
     @VisibleForTesting
     protected PeripheralKeymap mProcessor = new PeripheralKeymap(this, this);
 
+
+    @Override
+    public void onClick(View view) {
```
Comment (infinyte7) [inline AnkiDroid/src/main/java/com/ichi2/anki/Reviewer.java:112]: I tried to extract to ```Whiteboard.java``` but always getting error ```null reference``` but here working fine.

File: AnkiDroid/src/main/java/com/ichi2/anki/Reviewer.java [hunk 2]
```diff
@@ -120,6 +170,20 @@ protected void onCreate(Bundle savedInstanceState) {
         Timber.d("onCreate()");
         super.onCreate(savedInstanceState);
 
+        Button penColorWhite = (Button) findViewById(R.id.pen_color_white);
```
Comment (infinyte7) [inline AnkiDroid/src/main/java/com/ichi2/anki/Reviewer.java:173]: I tried to extract to Whiteboard.java but always getting error null reference but here working fine.

**Comment by infinyte7** [comment]
Now not able to open this.
```
2020-06-24 18:17:35.661 5905-5905/? E/libc: Access denied finding property "persist.vendor.sys.activitylog"
2020-06-24 18:17:37.523 5905-5905/com.ichi2.anki E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.ichi2.anki, PID: 5905
    java.lang.ExceptionInInitializerError
        at java.lang.Class.newInstance(Native Method)
        at android.app.AppComponentFactory.instantiateApplication(AppComponentFactory.java:50)
        at androidx.core.app.CoreComponentFactory.instantiateApplication(CoreComponentFactory.java:52)
        at android.app.Instrumentation.newApplication(Instrumentation.java:1120)
        at android.app.LoadedApk.makeApplication(LoadedApk.java:1061)
        at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5873)
        at android.app.ActivityThread.access$1200(ActivityThread.java:206)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1678)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6702)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:911)
     Caused by: java.lang.RuntimeException: Failed to initialize JaCoCo.
        at org.jacoco.agent.rt.internal_43f5073.Offline.getRuntimeData(Offline.java:41)
        at org.jacoco.agent.rt.internal_43f5073.Offline.getProbes(Offline.java:60)
        at com.ichi2.anki.AnkiDroidApp.$jacocoInit(Unknown Source:13)
        at com.ichi2.anki.AnkiDroidApp.<clinit>(Unknown Source:0)
        at java.lang.Class.newInstance(Native Method) 
        at android.app.AppComponentFactory.instantiateApplication(AppComponentFactory.java:50) 
        at androidx.core.app.CoreComponentFactory.instantiateApplication(CoreComponentFactory.java:52) 
        at android.app.Instrumentation.newApplication(Instrumentation.java:1120) 
        at android.app.LoadedApk.makeApplication(LoadedApk.java:1061) 
        at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5873) 
        at android.app.ActivityThread.access$1200(ActivityThread.java:206) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1678) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loop(Looper.java:193) 
        at android.app.ActivityThread.main(ActivityThread.java:6702) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:911) 
     Caused by: java.io.FileNotFoundException: /jacoco.exec (Read-only file system)
        at java.io.FileOutputStream.open0(Native Method)
        at java.io.FileOutputStream.open(FileOutputStream.java:308)
        at java.io.FileOutputStream.<init>(FileOutputStream.java:238)
        at org.jacoco.agent.rt.internal_43f5073.output.FileOutput.openFile(FileOutput.java:68)
        at org.jacoco.agent.rt.internal_43f5073.output.FileOutput.startup(FileOutput.java:50)
        at org.jacoco.agent.rt.internal_43f5073.Agent.startup(Agent.java:129)
        at org.jacoco.agent.rt.internal_43f5073.Agent.getInstance(Agent.java:54)
        at org.jacoco.agent.rt.internal_43f5073.Offline.getRuntimeData(Offline.java:39)
        at org.jacoco.agent.rt.internal_43f5073.Offline.getProbes(Offline.java:60) 
        at com.ichi2.anki.AnkiDroidApp.$jacocoInit(Unknown Source:13) 
        at com.ichi2.anki.AnkiDroidApp.<clinit>(Unknown Source:0) 
        at java.lang.Class.newInstance(Native Method) 
        at android.app.AppComponentFactory.instantiateApplication(AppComponentFactory.java:50) 
        at androidx.core.app.CoreComponentFactory.instantiateApplication(CoreComponentFactory.java:52) 
        at android.app.Instrumentation.newApplication(Instrumentation.java:1120) 
        at android.app.LoadedApk.makeApplication(LoadedApk.java:1061) 
        at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5873) 
        at android.app.ActivityThread.access$1200(ActivityThread.java:206) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1678) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loop(Looper.java:193) 
        at android.app.ActivityThread.main(ActivityThread.java:6702) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:911) 
```

**Comment by infinyte7** [comment]
I will convert it to draft. If anybody want to take this and continue then okay.

**Comment by david-allison-1** [comment]
@mikehardy The JaCoCo error appears related to #6541

File: AnkiDroid/src/main/java/com/ichi2/anki/Reviewer.java (hunk 2 above)
Comment (david-allison-1) [reply AnkiDroid/src/main/java/com/ichi2/anki/Reviewer.java:173]: `findViewById` on an activity will return different content than when run on a `View`.

**Pull request closed without merging**